### DISCUSSION TITLE
Difference between the open source and cloud version

### DISCUSSION THREAD

**Discussion by khanakia** [stack-auth/stack-auth#345] [Q&A]
Is there any comparison table between cloud and open source, such as what features are missing in open source compared to cloud? 

or which features are paid only like Saml, so etc.

**Comment by fomalhautb** [comment]
Feature-wise, the cloud version is exactly the same as the open-source version. The main difference and benefit of the cloud is that we handle all the infrastructure details like migration, downtime, security, latency, scaling, etc. for you so you can just use it with an API key.
